libs/audit-service/src/test/java/uk/gov/di/ipv/core/library/auditing/helpers/AuditExtensionsHelperTest.java

```diff
@@ -109,7 +109,7 @@ void getAuditRestrictedInheritedIdentityShouldReturnTheRightStuff() throws Excep
                 "[{\"personalNumber\":\"AB123456C\"}]",
                 OBJECT_MAPPER.writeValueAsString(restricted.socialSecurityRecord()));
         assertEquals(
-                "{\"device_information\":{\"encoded\":\"test_device_data\"}}",
+                "{\"encoded\":\"test_device_data\"}",
                 OBJECT_MAPPER.writeValueAsString(restricted.deviceInformation()));
```

Comment:
A bit odd that this has changed? Can we double-check that we haven't accidentally changed the format of the device information (by checking logs in dev, for example)

Reply:
Ahh yep, originally the `IPV_INHERITED_IDENTITY_VC_RECEIVED` restricted section looked like:
```
restricted: {
  deviceInformation: {
    device_information: {}
  }
}
```
but it should look like this instead:
```
restricted: {
  device_information: {}
}
```
Confirmed with Patrick that this was a bug and is fine to change :) (but I did forget to re-name the json property from deviceInformation -> device_information)

Reply:
Nice spot!